cmd/server/main.go

```diff
@@ -109,18 +110,35 @@ func main() {
 		klog.ErrorS(err, "failed to listen", "addr", addr, "proto", proto)
 		os.Exit(1)
 	}
+
 	opts := []grpc.ServerOption{
 		grpc.UnaryInterceptor(utils.UnaryServerInterceptor),
 	}
 
+	// register kms v1 server
 	s := grpc.NewServer(opts...)
```

Comment:
Having explicit name for the method will make it more readable, so make this `NewKMSv1Server`

Reply:
Isn't this from the `grpc` package? Perhaps you meant a method that returns `KeyManagementServiceServer`? In that case, I have updated it.